client/my-sites/site-settings/search.jsx

```diff
@@ -129,16 +159,23 @@ class Search extends Component {
 			widgetURL = untrailingslashit( widgetURL ) + '/widgets.php';
 		}
 
+		let renderedSettings = null;
+		if ( siteIsJetpack ) {
+			renderedSettings = this.renderJetpackSettings();
+		} else {
+			renderedSettings = this.renderWPComSettings();
+		}
+
 		return (
 			<div>
 				{ siteId && <QueryJetpackConnection siteId={ siteId } /> }
 
-				<SectionHeader label={ translate( 'Search' ) } />
+				<SectionHeader label={ translate( 'Jetpack Search' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Jetpack search' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
👍